src/lib/view.js

```diff
@@ -175,6 +175,9 @@ export default class View extends Component {
     }
 }
 
+/**
+ * @static @type {number | null} */
```

Comment:
would `number?` be the same?

Reply:
Yes, changed to ?number.

Note that we don't have strict null checks in typescript (we could enable that later), it was closure who was complaining about this one.